### ISSUE TITLE
Enemies(Graphics)

### ISSUE THREAD

**Issue by Aakburns** [Secretchronicles/TSC#272]
This game really is sort of all over the place.  The art work is in general, good, but can anyone see what's wrong here?
![capture1](https://cloud.githubusercontent.com/assets/284450/5333552/49a34326-7e40-11e4-8a30-0e033934704d.png)

The creature on the right looks very clean and 3d, the one on the left looks flat and 2d with less detail.

Basically, it's very inconsistent.  We need to pick a style and stick with it.  The characters should all look very 2d or 3d.

**Comment by Bugsbane** [comment]
As the creator of the image on the right (I call it "biteme"), personally I go for a 2D+shading look. I really don't try to replicate 3D at all, but I understand the overall point about consistency in art style, which I wholeheartedly agree on. Currently the visual style *is* all over the place, which is why I started the style guide, not only as a set of rules, but as a resource for new, less experienced artists to learn from. My hope is that we'll add to it over time, with the decisions by whichever experienced artists work on the project.

**Comment by Quintus** [comment]
I agree with you on this. Though I think replacing graphics like Flyon should be done later, we have more pressing things to resolve currently. On the long term, giving all graphcis a similar look would be achievement of a dream.

Vale,
Quintus
